miio/integrations/roidmi/vacuum/roidmivacuum_miot.py

```diff
@@ -580,7 +578,6 @@ def cleaning_summary(self) -> RoidmiCleaningSummary:
         return RoidmiCleaningSummary(
             {
                 prop["did"]: prop["value"] if prop["code"] == 0 else None
```

Comment:
```suggestion
                prop["did"]: prop["value"] if prop["code"] == 0 else None
                # max_properties limmit to 10 to avoid "Checksum error" messages from the device.
```